compiler/rustc_middle/src/ty/context.rs

```diff
@@ -967,7 +967,7 @@ pub struct GlobalCtxt<'tcx> {
     export_map: ExportMap<LocalDefId>,
 
     pub(crate) untracked_crate: &'tcx hir::Crate<'tcx>,
-    pub(crate) definitions: &'tcx Definitions,
+    pub(crate) definitions: Definitions,
```

Comment:
This is the only change I can think of that could caused the regression if it isn't just noise.